cmd/gokr-packer/packer.go

```diff
@@ -61,6 +71,14 @@ var (
 		"gokrazy",
 		"host name to set on the target system. Will be sent when acquiring DHCP leases")
 
+	// TDOD: Generate unique hostname on bootstrap e.g. gokrazy-<5-10 random characters>?
+	useTLS = flag.String("tls",
+		"",
+		`TLS certificate for the web interface (-tls=<certificate or full chain path>,<private key path>).
+Use -tls=self-signed to generate a self-signed RSA4096 certificate using the hostname specified with -hostname. In this case, the certificate and key will be placed in ~/.config/gokrazy/<hostname>/.
```

Comment:
I’m not entirely sure it’s the best choice to have the packer (optionally) generate self-certificates. Could we instead rely on e.g. https://github.com/FiloSottile/mkcert, or some other tool, and just reference a documentation page outlining how to use that tool?

Reply:
I agree on referencing a documentation on how to create certificate to use with the `-tls=<certificate or full chain path>,<private key path>` option. But I think we should encourage using encryption by providing a "hands free" option here. Adding encryption to a development setup should be as easy and fast as possible (IMHO).

By the way: I should document the whole "certificate pinning" logic and how to use it with own certificates ;)

Reply:
Fair enough. Let’s add that documentation reference and keep the rest as-is, then.